Library/Homebrew/formula_auditor.rb

```diff
@@ -662,12 +662,6 @@ def audit_specs
       stable_url_version = Version.parse(stable.url)
       stable_url_minor_version = stable_url_version.minor.to_i
 
-      formula_suffix = stable.version.patch.to_i
```

Comment:
This should not be removed unless @Homebrew/core agrees the approach in this PR is needed/better.

Reply:
Note: I don't agree with changing the throttling scheme for formulae, and I think neither does @SMillerDev (given the 👍).

Reply:
Thanks @carlocab @SMillerDev. Yeh, in that case: let's strip this out please @reitermarkus.